conans/client/toolchain/cmake/ios.py

```diff
@@ -0,0 +1,133 @@
+        message("Using Conan toolchain through ${CMAKE_TOOLCHAIN_FILE}.")
+        # We are going to adjust automagically many things as requested by Conan
+        #   these are the things done by 'conan_basic_setup()'
+        set(CMAKE_EXPORT_NO_PACKAGE_REGISTRY ON)
+        # To support the cmake_find_package generators
+        {% if cmake_module_path -%}
+        set(CMAKE_MODULE_PATH {{ cmake_module_path }} ${CMAKE_MODULE_PATH})
+        {%- endif %}
+        {% if cmake_prefix_path -%}
+        set(CMAKE_PREFIX_PATH {{ cmake_prefix_path }} ${CMAKE_PREFIX_PATH})
+        {%- endif %}
+        # shared libs
+        {% if shared_libs -%}
+        message(STATUS "Conan toolchain: Setting BUILD_SHARED_LIBS= {{ shared_libs }}")
+        set(BUILD_SHARED_LIBS {{ shared_libs }})
+        {%- endif %}
+
+        # C++ Standard
+        {% if cppstd -%}
+        message(STATUS "Conan C++ Standard {{ cppstd }} with extensions {{ cppstd_extensions }}}")
+        set(CMAKE_CXX_STANDARD {{ cppstd }})
+        set(CMAKE_CXX_EXTENSIONS {{ cppstd_extensions }})
+        {%- endif %}
+        # Install prefix
+        {% if install_prefix -%}
+        set(CMAKE_INSTALL_PREFIX "{{install_prefix}}" CACHE STRING "" FORCE)
+        {%- endif %}
+
+        # iOS stuff
+        # conan vars
+        set(CONAN_SETTINGS_HOST_ARCH "{{host_architecture}}")
+        set(CONAN_SETTINGS_HOST_OS "{{host_os}}") # CMAKE_SYSTEM_NAME
+        set(CONAN_SETTINGS_HOST_OS_VERSION "{{host_os_version}}") # SDK_VERSION
+        set(CONAN_SDK_NAME "{{host_sdk_name}}")
+        # TODO: add logic to calc the deployment target
+        set(CONAN_SETTINGS_HOST_MIN_OS_VERSION "{{host_os_min_version}}") # DEPLOYMENT TARGET
+
+        # set cmake vars
+        set(CMAKE_SYSTEM_NAME ${CONAN_SETTINGS_HOST_OS})
+        set(CMAKE_SYSTEM_VERSION ${CONAN_SETTINGS_HOST_OS_VERSION})
+        set(DEPLOYMENT_TARGET ${CONAN_SETTINGS_HOST_MIN_OS_VERSION})
+        # Set the architectures for which to build.
+        set(CMAKE_OSX_ARCHITECTURES ${CONAN_SETTINGS_HOST_ARCH})
+        # Setting CMAKE_OSX_SYSROOT SDK, when using Xcode generator the name is enough
+        # but full path is necessary for others
+        set(CMAKE_OSX_SYSROOT "${CONAN_SDK_NAME}")
+        if(NOT DEFINED CMAKE_XCODE_ATTRIBUTE_DEVELOPMENT_TEAM)
+          set(CMAKE_XCODE_ATTRIBUTE_DEVELOPMENT_TEAM "123456789A" CACHE INTERNAL "")
+        endif()
+    """)
+
+    def __init__(self, conanfile, build_type=None, **kwargs):
+        super(CMakeiOSToolchain, self).__init__(conanfile, build_type=build_type, **kwargs)
+        self.build_type = build_type or self._conanfile.settings.get_safe("build_type")
+        self.host_architecture = self._get_architecture()
+        self.host_os = self._conanfile.settings.get_safe("os")
+        self.host_os_version = self._conanfile.settings.get_safe("os.version")
+        self.host_sdk_name = self._get_sdk_name(self.host_architecture)
+        self.host_os_min_version = "9.0"
```

Comment:
This constant "9.0" is a bit concerning, please add at least an explanation, a warning or something.